modules/install-sno-generating-the-install-iso-manually.adoc

```diff
@@ -109,6 +109,14 @@ bootstrapInPlace:
 pullSecret: '<pull_secret>' <8>
 sshKey: |
   <ssh_key> <9>
+spec:
```

Comment:
There is no `spec` part on top of capabilities. The correct format is similar as below:
```
baselineCapabilitySet: v4.11 <10>
additionalEnabledCapabilities: <11>
  - CSISnapshot
  - Insights
  - Storage
```

Reply:
Thanks for the review @jinyunma I will correct that. Does the comment also apply to the change at line 210 (below)? I presume it does

Reply:
yes, seems in both sections.

Reply:
Thanks. I removed `spec` and adjusted the indentation for the subsequent lines